src/Resources/OAuth/Contacts/ContactIdentifiersResource.php

```diff
@@ -73,4 +73,25 @@ public function link(string $contactIdentifierValue, string $contactUuid): Conta
 
         return $mapper->map($response->getData());
     }
+
+    public function unlink(string $contactIdentifierValue): ContactIdentifier
+    {
+        $response = $this->client->put("$this->resourceUri/unlink", [
+            "contact_identifier_value" => $contactIdentifierValue,
+        ]);
+
+        $mapper = new ContactIdentifierMapper();
+
+        return $mapper->map($response->getData());
+    }
+
+
+    public function delete(string $contactIdentifierValue)
```

Comment:
Missing return type

Reply:
Return type is null, hoe wil je dat returnen?

Reply:
Hmm, ik zou het dan doen met een phpdoc:
```php
/**
* @return null
*/
```

Want volgensmij als je nu PHPStan draait, dan faalt deze hier ook.

Reply:
Bij mij slaagt stan met en zonder return type. Maar nu toegevoegd hoor